templates/_helpers.tpl

```diff
@@ -23,7 +23,7 @@
   version: 2.6.6
 # Used in CI
 - name: protoc
-  version: 3.19.1
+  version: 21.5
```

Comment:
It is 21.5 (and not 3.21.5).
They changed the release tags from some reason while protoc itself keeps reporting 3.21.5 when you run `protoc --version`).